mkosi/config.py

```diff
@@ -246,6 +247,10 @@ class DocFormat(StrEnum):
     pandoc   = enum.auto()
     system   = enum.auto()
 
+    @classmethod
+    def all(cls) -> list["DocFormat"]:
+        return [cls.man, cls.pandoc, cls.markdown, cls.system]
```

Comment:
You can just do `list(DocFormat)` no?

Reply:
No, because this would include `auto` and this also encodes their order, which otherwise would be by the string value, i.e. lexicographical. Added a comment to that extent.